kolibri/plugins/device_management/assets/src/modules/deviceInfo/handlers.js

```diff
@@ -12,6 +12,10 @@ export function getDeviceInfo() {
     const data = response.entity;
     data.server_time = new Date(data.server_time);
     data.content_storage_free_space = bytesForHumans(data.content_storage_free_space);
+
+    if (response.headers.Server.includes('0.0.0.0')) data.server_type = 'Kolibri internal server';
+    else data.server_type = response.headers.Server;
+
```

Comment:
What are the possible values of this header?

Reply:
I was wondering about the same...?

Reply:
It depends on the used server: CherryPy, Apache, Nginx, each server shows a string with its name and the version. It would serve us to give the better helping instructions knowing the exact server type and version they have installed (if any). If they are running just  `kolibri start ...` it will show 'Kolibri internal server`

Reply:
@jredrejo it's a great idea! I can easily picture a simple user who's actually running IIAB or Rachel in future releases. This user will not be aware of the HTTP server that's in front of Kolibri.

Reply:
Agreed, sounds like a very useful piece of information! 👍